packages/amplify-graphql-auth-transformer/src/__tests__/rds-mutation-auth.test.ts

```diff
@@ -12,6 +16,20 @@ describe('Verify RDS Model level Auth rules on mutations:', () => {
   const ADMIN_UI_ADMIN_ROLES =
     '$util.qr($ctx.stash.put(\\"adminRoles\\", [\\"us-fake-1_uuid_Full-access/CognitoIdentityCredentials\\",\\"us-fake-1_uuid_Manage-only/CognitoIdentityCredentials\\"])';
 
+  const dbConnectionConfig: SqlModelDataSourceDbConnectionConfig = {
+    hostnameSsmPath: '/test/hostname',
+    portSsmPath: '/test/port',
+    usernameSsmPath: '/test/username',
+    passwordSsmPath: '/test/password',
+    databaseNameSsmPath: '/test/databaseName',
+  };
```

Comment:
can we abstract this out into a common helper since I've seen this being constructed in other places as well. Maybe we can combine this with the snippet below to construct a default <MySql/Postgres>Strategy.

Reply:
Yes, it's a good idea for test utilities; I'll take it on in the upcoming combine PR